packages/backend/src/services/ExcelService/ExcelService.ts

```diff
@@ -326,6 +346,27 @@ export class ExcelService {
                     );
 
                     const row = worksheet.addRow(rowObject);
+
+                    // Apply format expressions only (values already set via addRow)
```

Comment:
Did you try adding formatting in the columns instead of each cell?
The docs suggest we can:
```
// Apply styles to worksheet columns
ws.columns = [
  { header: 'Id', key: 'id', width: 10 },
  { header: 'Name', key: 'name', width: 32, style: { font: { name: 'Arial Black' } } },
  { header: 'D.O.B.', key: 'DOB', width: 10, style: { numFmt: 'dd/mm/yyyy' } }
];
```
https://github.com/exceljs/exceljs?tab=readme-ov-file#styles